docs/DEVGUIDE.md

```diff
@@ -209,252 +209,13 @@ export KUBECONFIG=/home/yippee/code/service-catalog/.kubeconfig
-status:
-  conditions:
-  - message: The instance was provisioned successfully
-    reason: ProvisionedSuccessfully
-    status: "True"
-    type: Ready
-```
-
-### Bind to the Instance
-
-Now that our `Instance` has been created, let's bind to it.  To do this, we
-create a new [`Binding`](../contrib/examples/walkthrough/ups-binding.yaml).
-
-    $ kubectl create -f contrib/examples/walkthrough/ups-binding.yaml
-    binding "ups-binding" created
-
-We can check the status of the `Instance` using `kubectl get`:
-
-    $ kubectl get bindings -n test-ns ups-binding -o yaml
-
-We should see something like:
-
-```yaml
-apiVersion: servicecatalog.k8s.io/v1alpha1
-kind: Binding
-metadata:
-  creationTimestamp: 2017-03-07T01:44:36Z
-  finalizers:
-  - kubernetes
-  name: ups-binding
-  namespace: test-ns
-  resourceVersion: "29"
-  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/bindings/ups-binding
-  uid: 9eb2cdce-02d7-11e7-8edb-0242ac110005
-spec:
-  instanceRef:
-    name: ups-instance
-  osbGuid: b041db94-a5a0-41a2-87ae-1025ba760918
-  secretName: my-secret
-status:
-  conditions:
-  - message: Injected bind result
-    reason: InjectedBindResult
-    status: "True"
-    type: Ready
-```
-
-Notice that the status has a ready condition set.  This means our binding is
-ready to use.  If we look at the secrets in our `test-ns` namespace in
-kubernetes, we should see:
-
-    $ kubectl get secrets -n test-ns
-    NAME                  TYPE                                  DATA      AGE
-    default-token-3k61z   kubernetes.io/service-account-token   3         29m
-    my-secret             Opaque                                2         1m
-
-Notice that a secret named `my-secret` has been created in our namespace.
-
-### Unbind from the Instance
-
-Now, let's unbind from the Instance.  To do this, we just delete the `Binding`
-that we created:
-
-    $ kubectl delete -n test-ns bindings ups-binding
-
-If we check the secrets in the `test-ns` namespace, we should see that the
-secret we were injected with has been deleted:
-
-    $ kubectl get secrets -n test-ns
-    NAME                  TYPE                                  DATA      AGE
-    default-token-3k61z   kubernetes.io/service-account-token   3         30m
-
-### Deprovision the Instance
-
-Now, we can deprovision the instance.  To do this, we just delete the `Instance`
-that we created:
-
-    $ kubectl delete -n test-ns instances ups-instance
-
-### Delete the broker
-
-When an administrator wants to remove a broker and the services it offers from
-the catalog, they can just delete the broker:
-
-    $ kubectl delete brokers ups-broker
-
-And we should see that all the `ServiceClass` resources that came from that
-broker were cleaned up:
-
-    $ kubectl get serviceclasses
-    No resources found
+If you choose third party resources storage, the helm chart will not launch an etcd
+server, and instruct the API server to store all resources in the Kubernetes cluster
```

Comment:
and instruct -> but will instead instruct

or some form that reads less awkwardly.